ceremonies/2020/root-x2.yaml

```diff
@@ -25,6 +28,7 @@ skip-lints:
   # This is a root, not a sub-CA, so these don't apply.
   - e_ext_authority_key_identifier_missing
   - e_ext_authority_key_identifier_no_key_identifier
+  - w_sub_ca_aia_missing
```

Comment:
Does `e_sub_ca_aia_missing` need to be removed? It's surprising to me that both warning and error versions of the same lint exist.

Reply:
[e_sub_ca_aia_missing](https://github.com/zmap/zlint/blob/71d5e4b1cbf7636331bbf1c839d24abb4a945633/v3/lints/cabf_br/lint_e_sub_ca_aia_missing.go#L25-L31) can be removed because it had an [ineffective date](https://github.com/zmap/zlint/blob/71d5e4b1cbf7636331bbf1c839d24abb4a945633/v3/util/time.go#L69) of Aug 20, 2020 and was superseded by [w_sub_ca_aia_missing](https://github.com/zmap/zlint/blob/71d5e4b1cbf7636331bbf1c839d24abb4a945633/v3/lints/cabf_br/lint_w_sub_ca_aia_missing.go). To note there is that the effective date is the same as e_sub_ca_aia_missing's ineffective date. Root X2 was issued on Sept 4, 2020 and would be subject to at least [BR version 1.7.1](https://github.com/zmap/zlint/blob/71d5e4b1cbf7636331bbf1c839d24abb4a945633/v3/util/time.go#L69).